src/third_party/dartino/object_memory_mark_sweep.cc

```diff
@@ -630,7 +630,6 @@ void OldSpace::validate_sweep(Chunk* chunk) {
     ASSERT(GcMetadata::all_mark_bits_are(object, size, alive ? 1 : 0));
     ASSERT(object->is_a_free_object() == !alive);
     if (*starts != GcMetadata::NO_OBJECT_START) {
-      ASSERT(*starts < GcMetadata::CARD_SIZE);
```

Comment:
Not sure I understand why this had to go?

Reply:
On 32 bit it was wrong, and on 64 bit it could never trigger.